cli/githubutil.py

```diff
@@ -138,6 +142,12 @@ def release_and_prepare_next_dev_cycle(
         prerelease=prerelease_suffix
     )
 
+    draft_name = draft_release_name_for_version(release_version)
+    draft_release = helper.draft_release_with_name(draft_name)
+    if draft_release:
+        verbose('cleaning up draft release {name}'.format(name=draft_release.name))
```

Comment:
verbose takes no arguments. I suppose you ment 'info()'

Reply:
are you sure, or am I missing something? https://github.com/gardener/cc-utils/blob/master/util.py#L121

Reply:
I stand corrected. In that case there are only nits from my point of view

Reply:
ok, thanks